L73-java-binderchannel/wireformat.md

```diff
@@ -201,7 +202,13 @@ ungraceful shutdown will occur, with the status code CANCELLED. The remote
 transport will be notified, cancelling all ongoing remote calls on the server
 side.
 
+## Flow Control
+
+BinderTransport now supports flow control, aiming to keep use of the process transaction buffer to at most 128k. Each transaction we send adds to an internal count of unacknowledged outbound data. If that exceeds 128k, we’ll stop sending transactions until we receive an acknowledgement message from the transport peer. Each transport sends an acknowledgement for each 16k received, which should avoid blocking the transport most of the time.
```

Comment:
Probably worth to mention that we want to do this because Android binder implementation has an internal buffer with fixed size?

Reply:
Done